runtime/src/weights/pallet_multisig.rs

```diff
@@ -0,0 +1,273 @@
+// Copyright 2024, Horizen Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO: Autogenerate with remote machine and add details + executed command
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+#![allow(missing_docs)]
+
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use core::marker::PhantomData;
+
+/// Weight functions needed for `pallet_multisig`.
+pub trait WeightInfo {
+    fn as_multi_threshold_1(z: u32, ) -> Weight;
+    fn as_multi_create(s: u32, z: u32, ) -> Weight;
+    fn as_multi_approve(s: u32, z: u32, ) -> Weight;
+    fn as_multi_complete(s: u32, z: u32, ) -> Weight;
+    fn approve_as_multi_create(s: u32, ) -> Weight;
+    fn approve_as_multi_approve(s: u32, ) -> Weight;
+    fn cancel_as_multi(s: u32, ) -> Weight;
+}
+
```

Comment:
Ok that's a temp file but remove this useless trait from this file ... at least to shut up the warring :smile: